src/TableFunctions/TableFunctionObjectStorage.cpp

```diff
@@ -183,6 +183,10 @@ template class TableFunctionObjectStorage<HDFSClusterDefinition, StorageHDFSConf
 #endif
 template class TableFunctionObjectStorage<LocalDefinition, StorageLocalConfiguration>;
 
+#if USE_AVRO
```

Comment:
Why does this depend on `AVRO`?

Reply:
I have no answer why, but everything Iceberg-related in code under `#if USE_AVRO`. 'Why' is a question to ClickHouse Inc.